test/integration/user-management-api/service_plan_features_test.rb

```diff
@@ -44,15 +44,6 @@ def setup
     assert @service_plan.features.reload.include?(feat)
   end
 
-  test 'associate feature to service plan twice' do
```

Comment:
I guess this test is related to this controller?

https://github.com/3scale/porta/blob/1be51d3b3ff6d5f5e7f836d334c4b4a0082fc3cf/app/controllers/admin/api/featurings_base_controller.rb#L9-L12

So, what would happen after adding a PK? Maybe we need to add another test that would verify the behavior?

Reply:
Yeah, I think the new behavior is correct (I mean - I doubt someone really relied on the old behavior returning a 200 OK to adding the same feature twice)

But yes, it's just the matter of adding a test to know what is actually returned now (and having the test would confirm that the behavior is expected).

Reply:
Done!